Tests/Marketplace/Tests/marketplace_services_test.py

```diff
@@ -741,6 +755,72 @@ def test_prepare_release_notes_upgrade_version_dup(self, mocker, dummy_pack):
         assert task_status is True
         assert not_updated_build is False
 
+    @pytest.mark.parametrize('version, boolean_value', [('1.0.1', True), ('1.0.2', False)])
```

Comment:
Please test the case where it is in the changelog but it is part of aggregated version

Reply:
Done